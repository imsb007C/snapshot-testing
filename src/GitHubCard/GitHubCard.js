import React from "react"
import Card from "react-bootstrap/Card"

function GitHubCard() {
    return(
        <div className="card">
            <Card style={{width: "25em"}}>
                <Card.Img variant="top" src={"https://github.com/imsb007c.png"}/>
                <Card.Body>
                    <Card.Title>imsb007c</Card.Title>
                    <Card.Text>
                        My name is Alex Chen
                    </Card.Text>
                </Card.Body>
            </Card>
        </div>
    )
}
export default GitHubCard