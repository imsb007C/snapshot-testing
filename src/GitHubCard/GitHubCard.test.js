import GitHubCard from './GitHubCard'
import renderer from 'react-test-renderer'

test('rendera snapshot',() => {
    const tree = renderer.create(<GitHubCard />).toJSON()
    expect(tree).toMatchSnapshot()
})