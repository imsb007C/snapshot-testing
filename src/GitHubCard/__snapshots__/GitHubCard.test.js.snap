// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendera snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card"
    style={
      Object {
        "width": "25em",
      }
    }
  >
    <img
      className="card-img-top"
      src="https://github.com/imsb007c.png"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        imsb007c
      </div>
      <p
        className="card-text"
      >
        My name is Alex Chen
      </p>
    </div>
  </div>
</div>
`;
